src/clp_ffi_py/ir/native/PyFourByteDeserializer.cpp

```diff
@@ -61,43 +61,44 @@ PyMethodDef PyDeserializer_method_table[]{
 
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays)
 PyDoc_STRVAR(
-        cPyDeserializerDoc,
-        "Namespace for all CLP IR deserialization methods.\n\n"
+        cPyFourByteDeserializerDoc,
+        "Namespace for all CLP four byte IR deserialization methods.\n\n"
```

Comment:
```suggestion
        "Namespace for all CLP four-byte encoded IR deserialization methods.\n\n"
```

So it's consistent with what you wrote in the files below